### ISSUE TITLE
No comments in any files

### ISSUE THREAD

**Issue by adamryman** [col6y/codeday-fall-2014#1]
Why are there no comments?

**Comment by col6y** [comment]
Because long-term maintainability is not a focus when you only have 24 hours to write code.

**Comment by geniusninja88** [comment]
?

**Comment by col6y** [comment]
@geniusninja88 look on GitHub: https://github.com/col6y/codeday-fall-2014/issues/1

**Comment by geniusninja88** [comment]
That was a long time ago

**Comment by col6y** [comment]
@geniusninja88 Yes, but I ran across it again today and figured I wouldn't leave the issue unresponded to.
